instrumentation/opentelemetry-instrumentation-pymemcache/src/opentelemetry/instrumentation/pymemcache/__init__.py

```diff
@@ -115,7 +115,7 @@ def wrapper(wrapped, instance, args, kwargs):
 @_with_tracer_wrapper
 def _wrap_cmd(tracer, cmd, wrapped, instance, args, kwargs):
     with tracer.start_as_current_span(
-        _CMD, kind=SpanKind.INTERNAL, attributes={}
+        _CMD, kind=SpanKind.CLIENT, attributes={}
```

Comment:
I believe the span name should be a low-cardinality value representing the executed statement.

Reply:
I would also add `db.name` as a span attribute to be consistent with the other instrumentations.

Reply:
The span name for all pymemcache commands is `memcached.command`. Regarding `db.name` there is `db.system` which is set to [here](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/99d6dfcb4f68705d43120a5cd5a2c7e677ceaed6/instrumentation/opentelemetry-instrumentation-pymemcache/src/opentelemetry/instrumentation/pymemcache/__init__.py#L167). Do you think we need `db.name` set to same as well?

Reply:
> The span name for all pymemcache commands is memcached.command

I'm not too familiar in how to use pymemcache, but does that mean there is no "command" or "statement" that is executed?

> Do you think we need db.name set to same as well?

Yes, we have `db.name` in other instrumentations so we should be consistent.

Reply:
It has commands. Yes we should be using them instead of hard-coded `memcached.command` for span name. I don't think `db.name` is applicable for memcached.